javascript/javascript1/week3/script.js

```diff
@@ -16,10 +16,85 @@ function removeName(removalName) {
   for (let i = 0; i < names.length; i++) {
     if (names[i] === "Ahmad") {
       names.splice(i, 1);
+      break;
     }
   }
 }
 removeName("Ahmad");
 // Code done
 
 console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+// ######  When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function travelTimeCalc(travelInformation) {
+  return travelInformation.destinationDistance / travelInformation.speed;
+}
+
+const travelTime = travelTimeCalc(travelInformation);
```

Comment:
You don't really use line 40 - 41, also it would better if you print something like `3 hours and 34 minutes`, right now the console looks like `8.64`.